opflexagent/utils/port_managers/async_port_manager.py

```diff
@@ -112,7 +112,7 @@ def schedule_update(self, device_ids=None):
         device_ids = set(device_ids or [])
         LOG.debug('Update initially scheduled for port ids %s', device_ids)
         # See if more ports need to be updated due to request timeout
-        for request in self.pending_requests.get_requests():
+        for request in list(self.pending_requests.get_requests()):
```

Comment:
Wonder why do we need to put list() around those returned by values() API here and below.....The values() API should already return a list even in python3 according to this,
https://www.tutorialspoint.com/python3/python_dictionary.htm